networks/meld/tokens/0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653.json

```diff
@@ -0,0 +1,15 @@
+{
+    "CMCslug": "",
+    "decimals": 18,
+    "mldId": "meld-mape-0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653",
+    "orgName": "MAPE",
+    "orgUrl": "",
+    "tokenAddress": "0xC89D6b70DcBD4DDf9e5E474Bb529d3F49Ef1c653",
+    "tokenColorTheme": 1,
```

Comment:
Color 1 is red, while this icon looks more blue to me.

Reply:
this was requested by the creator... I also find it weird, but let's keep it red